src/components/progress-circle/progress-circle.ts

```diff
@@ -250,8 +250,8 @@ function clamp(v: number) {
  * Returns the current timestamp either based on the performance global or a date object.
  */
 function now() {
-  if (window.performance && window.performance.now) {
-    return window.performance.now();
+  if (typeof performance !== 'undefined' && performance.now) {
```

Comment:
The point of the PR was to remove the references to `window`. Just doing `if (performance && performance.now)` would throw an exception if the client doesn't have the `performance` object.